cezo_fl_main.py

```diff
@@ -98,6 +100,12 @@ def prepare_settings_underseed(args, device):
     return model, criterion, optimizer, grad_estimator, accuracy_func
 
 
+def replace_penultimate_layer(model):
+    penultimate_layer = model.model.decoder.layers[-2]
+    penultimate_layer.fc1 = QuantizedLinearLayer(penultimate_layer.fc1)
+    penultimate_layer.fc2 = QuantizedLinearLayer(penultimate_layer.fc2)
```

Comment:
1. you can put this util func into `quantized_layer.py`.
2. Add type annotation and docstring
3. I feel you need to add a unit test to show the `QuantizedLinearLayer` is indeed applied